### ISSUE TITLE
[1.12.2] [HIGH PRIORITY] Chunk data not saving correctly? Tile-entities not being saved.

### ISSUE THREAD

**Issue by Zilacon** [Teneted/Tenet#197]
Both of my servers are experiencing very strange issues that have to be related to nbt data being saved or region data or something.

Placing a cable down, and leaving the chunk and coming back, will change it to a different type of cable if its part of the same forgename.

For example, if you place down a insulated copper cable from IC2 Classic and leave the chunk and come back, it changes to a uninsulated copper cable. Both cables are named IC2_ITEMCABLE

IronChests have the same issues, place any type of chest and place items in it and leave the chunk and come back they will change to different types of chests.

The rotation of t-e's arent being saved either. You can place it facing one way and when you leave and come back it will be facing another way. I saw a wall of chests which were placed facing the open, and when you come back they are all facing the wall.

This issue is also happening with non-modded vanilla chests and other t-e's.

Basically its happening with anything that has any form of data it seems.

**Comment by kaziu687** [comment]
Did your server crash? Maybe it's the same like #195

**Comment by Zilacon** [comment]
No, No crash. This is happening in realtime as chunks load/unload.

**Comment by Zilacon** [comment]
@Mgazul Not sure if related, but this is popping up.
```
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.core.tile.TileEngineRedstone_BC8 at BlockPos{x=2615, y=96, z=4051} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TilePump at BlockPos{x=2614, y=96, z=4051} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4050} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4049} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4048} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2475, y=69, z=4319} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2610, y=97, z=4040} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileTank at BlockPos{x=2610, y=97, z=4044} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2611, y=84, z=4036} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4044} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=86, z=4033} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.core.tile.TileEngineRedstone_BC8 at BlockPos{x=2613, y=86, z=4033} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4042} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=84, z=4036} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2613, y=84, z=4036} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2611, y=84, z=4038} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=84, z=4034} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2612, y=84, z=4036} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4046} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileTank at BlockPos{x=2610, y=96, z=4044} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2610, y=96, z=4042} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileTank at BlockPos{x=2610, y=98, z=4039} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=85, z=4033} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.core.tile.TileEngineRedstone_BC8 at BlockPos{x=2611, y=97, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2610, y=97, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileTank at BlockPos{x=2610, y=97, z=4039} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=84, z=4033} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4045} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4043} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=84, z=4035} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4047} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2611, y=84, z=4037} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileDistiller_BC8 at BlockPos{x=2610, y=96, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.factory.tile.TileTank at BlockPos{x=2610, y=96, z=4039} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2614, y=96, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2613, y=96, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2612, y=96, z=4041} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2610, y=96, z=4043} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.core.tile.TileEngineRedstone_BC8 at BlockPos{x=2609, y=96, z=4043} when reading from NBT
[07:04:35 WARN]: [lib.tile] Unknown owner (null) for class buildcraft.transport.tile.TilePipeHolder at BlockPos{x=2611, y=85, z=4038} when reading from NBT
```

**Comment by Zilacon** [comment]
Using latest version, problem is not fixed.

**Comment by Zilacon** [comment]
![Screenshot_498](https://user-images.githubusercontent.com/3002248/78739993-5cb1a000-790a-11ea-91c9-554fc7bdf4fa.png)

**Comment by Mgazul** [comment]
The most recent commit should be fixed

**Comment by Zilacon** [comment]
The amount of players reporting T-E's disappearing has dropped, but theres still reports of T-E's inventories being wiped.

Chests, ME Drives, Smeltery, etc are still being effected by having their items or liquids deleted.
